commands/generate-key.js

```diff
@@ -24,8 +44,17 @@ module.exports = {
             return;
         }
 
-        const keyPair = KeyPair.fromRandom('ed25519');
-        await keyStore.setKey(argv.networkId, argv.accountId, keyPair);
-        console.log(`Generated key pair with ${keyPair.publicKey} public key`);
+        let publicKey, accountId;
+        if (argv.seedPhrase) {
+            const result = parseSeedPhrase(argv.seedPhrase, argv.seedPath);
+            publicKey = result.publicKey;
```

Comment:
Is this a string or `PublicKey`?

Reply:
it's a string.

Reply:
https://github.com/near/near-seed-phrase/blob/master/index.js#L18

Reply:
seems like makes sense to handle this in middleware, similar to Ledger:
https://github.com/near/near-cli/blob/c6870c5f30799cb8cdcdc55046edba792d37daa6/middleware/ledger.js#L46